{"name": "Получение ошибки при авторизации пользователя с неверными email и паролем.", "status": "passed", "description": "Тест для проверки получения ошибки при авторизации пользователя с неверными обязательными даннами.", "steps": [{"name": "Подготовить неверные учетные данные", "status": "passed", "start": 1748676351729, "stop": 1748676351729}, {"name": "Отправить запрос на логин", "status": "passed", "start": 1748676351729, "stop": 1748676351918}, {"name": "Проверить код и сообщение об ошибке", "status": "passed", "start": 1748676351918, "stop": 1748676351918}], "parameters": [{"name": "credentials_modifier", "value": "<function TestAuth.<lambda> at 0x000001ED1A2FFE20>"}, {"name": "expected_message", "value": "'email or password are incorrect'"}], "start": 1748676351729, "stop": 1748676351918, "uuid": "39fb41ce-5c14-4511-97e1-839c44040147", "historyId": "77109ab4fc56c22bd2a5f549f28cd665", "testCaseId": "7f7b48c0611723ea675fab986352bc5e", "fullName": "tests.test_user_login.TestAuth#test_login_invalid_credentials_error_received", "labels": [{"name": "suite", "value": "Авторизация пользователя."}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "DESKTOP-G7OKOON"}, {"name": "thread", "value": "832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}], "links": [{"type": "link", "url": "https://code.s3.yandex.net/qa-automation-engineer/python-full/diploma/api-documentation.pdf", "name": "API документация"}]}